src/components/shareable-url-modal/published-view/published-view.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import classnames from 'classnames';
+import UrlBox from '../url-box/url-box';
+import Button from '../../ui/button';
+import Dropdown from '../../ui/dropdown';
+import MenuOption from '../../ui/menu-option';
+
+import { getFilteredPlatforms, handleResponseUrl } from '../utils';
+
+const PublishedView = ({
+  hostingPlatformLocalStorageVal,
+  hostingPlatforms,
+  onChange,
+  onCopyClick,
+  onRepublishClick,
+  platform,
+  showCopied,
+}) => {
+  const platformsKeys = Object.keys(hostingPlatformLocalStorageVal);
```

Comment:
```suggestion
  const hostedPlatformKeys = Object.keys(hostingPlatformLocalStorageVal);
```

Reply:
does - Object.keys(hostingPlatformLocalStorageVal); return ["aws", "azure", "gcp"] from the localStorage -
{
   aws: {
       bucket_name: ...,
       platform: 'aws',
       endpoint: '....',
   },
   gcp: {....},
   azure: {....}
}
?

Also, in localStorage do we have a parent to the above object ? Like do we store it under kedro-viz ?
kedro-viz: {
....
hostedPlatforms: {
aws: {
       bucket_name: ...,
       platform: 'aws',
       endpoint: '....',
   },
   gcp: {....},
   azure: {....}
}
}

Reply:
hey so 

- does - Object.keys(hostingPlatformLocalStorageVal); return ["aws", "azure", "gcp"] from the localStorage ? 
yes it's correct
- Also, in localStorage do we have a parent to the above object ? Like do we store it under kedro-viz ? 
yes so this one live under `KedroViz-sharable-url'`, like other things we stored in localStorage
```
export const localStorageName = 'KedroViz';
export const localStorageFlowchartLink = 'KedroViz-link-to-flowchart';
export const localStorageMetricsSelect = 'KedroViz-metrics-chart-select';
export const localStorageRunsMetadata = 'KedroViz-runs-metadata';
export const localStorageSharableUrl = 'KedroViz-sharable-url';
```